### ISSUE TITLE
Create "summer hack" project list, supporting resources

### ISSUE THREAD

**Issue by gregmarra** [the-blue-alliance/the-blue-alliance#1191]
Identify a list of projects people might contribute towards over the summer. Probably small projects like "fix this typo" and large projects like "build this new feature".

Projects should be reasonably well laid out, and have a clear "point of contact" of an existing contributor if the person who picks them up has questions.

I'm going to start applying "Help Wanted" to these, including this one :)

Tips:
https://18f.gsa.gov/2015/04/03/how-to-welcome-new-coders-to-a-civic-hackathon/

**Comment by phil-lopreiato** [comment]
Some ideas/things that have been kicking around for a while:
 - [ ] myTBA support on event & team pages (manage favorite/notification settings)
 - [ ] Implement 'Final Results' notifications - show the final status of a team@event
 - [ ] Some extension of #576 would be cool - have the ability to add "Pit Display"-like thing to GameDay instead of webcast: show next/last X matches, live-updating rankings
 - [ ] iOS support for push notifications
 - [ ] Sitevar for FRCAPI status: dict with boolean for each endpoint, check/uncheck each in admin panel
 - [ ] Disable FRCAPI on a per-event basis (same dict as above)
 - [ ] Make event alliances endpoint cronjob run more frequently (every 5 mins?), but only actually hit that endpoint that frequently on the final day of the event. This means we don't have to wait half an hour for notifications to go out. 
 - [ ] Infer event alliances from quarterfinal schedule
 - [ ] Break big json properties (alliances, rankings, etc.) out into their own models? Allow for easier querying/updating
 - [ ] Fix admin broadcast. I'm guessing the query is probably too big with >5,000 clients. Datastore queries should be batched, even though gcm should be able to handle that many
 - [ ] Server side support for https://github.com/the-blue-alliance/the-blue-alliance-android/issues/418
 - [ ] Moar insights, and add them to API
 - [ ] Internal support for DQs and surrogates in matches (and account for them in OPR, API, schedule view)

**Comment by synth3tk** [comment]
We could "assign" the point of contact to each of these issues so they can easily see who it is, as well as add the "Help Wanted" tag.

Also, add Gameday 2 to the list? It seems to have stagnated, so I don't know where we stand on that.

**Comment by fangeugene** [comment]
Good point about GameDay 2. I have ideas for cleaning up the interface (no more drag/drop because that's clunky) but keeping the same functionality. Unsure if building off of Greg's GameDay2 branch that uses React is the right thing to do though.

**Comment by gregmarra** [comment]
I would probably abandon my GD2 branch and refactor gd1 in place.

**Comment by synth3tk** [comment]
Sounds good.
